ui-components/src/locales/general.json

```diff
@@ -867,7 +870,10 @@
         "createAccountParagraph": "Creating an account will save your information for future applications, and you can check the status of this application anytime.",
         "imdone": "No thanks, I'm done.",
         "browseMore": "Browse more listings",
-        "print": "View submitted application and print a copy."
+        "print": "View submitted application and print a copy.",
+        "applicationReceived": "Application  \nreceived",
```

Comment:
We don't need to think about this here, but I'm not sure how this would work with translations. It's probably a bigger question.

Reply:
Good point, the available space may be a little harder to figure out in translations that are more verbose.